README.md

```diff
@@ -0,0 +1,280 @@
+# TaleTime
+
+Children may discover a novel way to listen to audiobooks with the help of the revolutionary Android and iOS software TaleTime.
+
+In addition to offering parents the chance to record the stories themselves, we also provide professionally recorded audiobooks. Recording is incredibly practical and, naturally, is something that anyone with a little understanding of technology can perform. Children at TaleTime can also decide how the stories will develop on their own.
+
+<!-- <div class="d-lg-none">
+    <p class="lead">
+        <a href="app">Test out our online demo!</a>
+    </p>
+</div> -->
+*[Test out our online demo](https://taletime-2022.web.app/#/)*
+
+## Write a new Story
+
+TaleTime depends on authors producing new works! The information below is for you if you want to help grow the application.
+
+To become a taletime story-teller, all you need is to register an account, create a 'story-teller' profile and start recording as many stories as you wish. 
+
+
+## Contributing
+
+Everyone may contribute to improving TaleTime as it is an open source project.
+The entire source code is available on [GitHub](https://github.com/TaleTime/TaleTime_2/tree/main/taletime), where you can also learn more about contributing.
+
+Please ensure that your code adheres to the same style as the existing codebase whether adding a new feature or resolving a problem, and make an effort to write your code as simply as you can.
+Additionally, test your modifications before committing them, and only do so if they don't damage anything.
+You must use English for commenting.
+
+## Installation
+
+### Flutter
+
+Download current [Flutter Version](https://docs.flutter.dev/get-started/install) and follow the installation guide for your OS.
+Here you will install Flutter and Android Studio, you will update your path Variable to use commands more easily inside the shell and configure your Android Emulator.
+The process is well documented and the command "flutter doctor" helps you find and solve any problems you might encounter. 
+
+#### Disclaimer
+
+During installation on Windows it might be necessary to update the included dart, use "flutter pub upgrade --major-versions" if needed.
+For any other encountered error you can use [Google](https://www.google.de/), the Flutter community is very active and helpful.
+
+### IDE
+
+#### Android Studio
+
+We recommend using [Android Studio](https://developer.android.com/studio) because it is an integrated development environment for Android projects.
+It comes with all tools required for developing and configuring your emulators.
+
+While already being configured for Android projects, the IDE is still missing support for Flutter.
+But this can simply be added by installing the [Flutter Plugin](https://plugins.jetbrains.com/plugin/9212-flutter).
```

Comment:
This can easily be added by installing the [Flutter Plugin] (https://plugins.jetbrains.com/plugin/9212-flutter).